RetailCoder.VBE/UI/Settings/GeneralSettings.xaml

```diff
@@ -137,7 +137,11 @@
                             Command="{Binding ShowLogFolderCommand}"/>
                     </StackPanel>
 
-                    <Label Content="{Resx ResxName=Rubberduck.UI.RubberduckUI, Key=GeneralSettings_HotkeysLabel}" FontWeight="SemiBold" />
+                    <DockPanel>
+                        <Label DockPanel.Dock="Left" Content="{Resx ResxName=Rubberduck.UI.RubberduckUI, Key=GeneralSettings_HotkeysLabel}" FontWeight="SemiBold" />
+                        <Label DockPanel.Dock="Right" Content="{Resx ResxName=Rubberduck.Ui.RubberduckUI, Key=GeneralSettings_HotkeyRestrictionLabel}"/>
```

Comment:
I'm not sure we should dock side-by-side. Also from a UX perspective we should consider showing the Restrictions only when relevant

Reply:
I couldn't come up with a way to conditional display a restriction message.

Reply:
```
if (settings.Any(s =>!s.IsValid())) 
{
    // display message
}
```

Reply:
^^ make that a `bool` property getter in the VM, and then bind it to the label's `Visibility` property with a `BoolToVisibilityConverter`.